<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ComAssist</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ComAssistant.qrc">
    <normaloff>:/ui/ComAssist.png</normaloff>:/ui/ComAssist.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBoxPort">
        <property name="title">
         <string>Serial Port Setting</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="label_port">
           <property name="text">
            <string>Port</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_port"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_baudrate">
           <property name="text">
            <string>Baudrate</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_baudrate">
           <item>
            <property name="text">
             <string>110</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>300</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_dataBits">
           <property name="text">
            <string>Data Bits</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_dataBits">
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_parity">
           <property name="text">
            <string>Parity</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox_parity">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Odd</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Even</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mark</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Space</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_stopBits">
           <property name="text">
            <string>Stop Bits</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBox_stopBits">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_flowType">
           <property name="text">
            <string>Flow Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="comboBox_flowType">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RTX/CTS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>XON/XOFF</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_receive">
        <property name="title">
         <string>Receive Setting</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="radioButton_receiveText">
             <property name="text">
              <string>Text</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_receiveHex">
             <property name="text">
              <string>Hex</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_feedLine">
           <property name="text">
            <string>Auto Feed Line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_displaySend">
           <property name="text">
            <string>Display Send</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_distplayTime">
           <property name="text">
            <string>Display Time</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_send">
        <property name="title">
         <string>Send Setting</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_sendText">
             <property name="text">
              <string>Text</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_sendHex">
             <property name="text">
              <string>Hex</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
           <item>
            <widget class="QCheckBox" name="checkBox_sendLoop">
             <property name="text">
              <string>Loop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_loopTime"/>
           </item>
           <item>
            <widget class="QLabel" name="label_ms">
             <property name="text">
              <string>ms</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_loopPerLine">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Loop Per Line</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_send">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_history">
        <item>
         <property name="text">
          <string>SendSpace1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SendSpace2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SendSpace3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SendSpace4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SendSpace5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SendSpace6</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit_receive">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QPlainTextEdit" name="plainTextEdit_send"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExist"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionASCIITable"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar_editor">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExist">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../ComAssistant.qrc">
     <normaloff>:/ui/start.png</normaloff>:/ui/start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../ComAssistant.qrc">
     <normaloff>:/ui/pause.png</normaloff>:/ui/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../ComAssistant.qrc">
     <normaloff>:/ui/stop.png</normaloff>:/ui/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../ComAssistant.qrc">
     <normaloff>:/ui/clear.png</normaloff>:/ui/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="icon">
    <iconset resource="../ComAssistant.qrc">
     <normaloff>:/ui/scan.png</normaloff>:/ui/scan.png</iconset>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="../ComAssistant.qrc">
     <normaloff>:/ui/options.png</normaloff>:/ui/options.png</iconset>
   </property>
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionASCIITable">
   <property name="text">
    <string>ASCII Table</string>
   </property>
  </action>
  <action name="actionCheckSum">
   <property name="text">
    <string>CheckSum</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../ComAssistant.qrc"/>
 </resources>
 <connections/>
</ui>
